app/celery/scheduled_tasks.py

```diff
@@ -222,6 +222,24 @@ def check_templated_letter_state():
             )
 
 
+@notify_celery.task(name="in-flight-to-inbox-sms")
+@statsd(namespace="tasks")
+def in_flight_to_inbox_sms():
+    _, notifications = sms_queue.expire_inflights()
```

Comment:
I would expect the API call to be sufficient to have automatically moved the notifications from the inflight back to the inbox, so there would be no notifications returned...